.github/workflows/test.yml

```diff
@@ -97,6 +97,26 @@ jobs:
           platforms: ${{ matrix.arch }}
       - run: docker run --platform=linux/${{ matrix.arch }} -v${HOME}/go/pkg/mod:/root/go/pkg/mod -v $PWD:$PWD -w $PWD -eCGO_ENABLED=${{ matrix.cgo_enabled }} -eDD_APPSEC_WAF_TIMEOUT=$DD_APPSEC_WAF_TIMEOUT golang go test -v -count=10 -shuffle=on ./...
 
+  windows-other:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch: ["386"]
+        cgo_enabled: [ "0", "1" ] # test it compiles with and without the cgo
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          cache: true
+      - name: go test
+        shell: bash
+        run: |
+          # Install gotestsum
+          env GOBIN=$PWD go install gotest.tools/gotestsum@latest
+          # Run the tests with gotestsum
+          env GOARCH=${{matrix.arch}} CGO_ENABLED=${{ matrix.cgo_enabled }} ./gotestsum -- -v -count=10 -shuffle=on ./...
```

Comment:
`windows-latest` is running on amd64 right ? How do you want to run 386 tests on this ?